lib/main/atom/atomConfig.ts

```diff
@@ -17,13 +17,19 @@ class Config {
             type: 'boolean',
             default: false
         },
+        modulePathToProjectRoot: {
+            title: 'Show module path suggestion relative to project root.',
+            type: 'boolean',
+            default: true
+        },
```

Comment:
I will merge. But will separately make the default `false` post merge as I personally prefer *relative* to file (and not relative to root). 

Also people might be confused by project (here you mean `atom` project but vs developers might think `tsconfig` project).